pom.xml

```diff
@@ -0,0 +1,172 @@
+        <resources>
+            <resource>
+                <directory>src/main/assembly/dist</directory>
+                <targetPath>${project.build.directory}/version</targetPath>
+                <includes>
+                    <include>version</include>
+                </includes>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.scalatest</groupId>
+                <artifactId>scalatest-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration combine.children="append">
+                    <excludes combine.self="override">
+                        <exclude>src/main/assembly/dist/extern/*.xsd</exclude>
+                        <exclude>src/main/assembly/dist/extern/**/*.xsd</exclude>
+                    </excludes>
+                    <includes>
+                        <include>pom.xml</include>
+                        <include>src/test/scala/**</include>
+                        <include>src/main/assembly/**/*.xsd</include>
+                        <include>src/main/assembly/**/*.xml</include>
+                    </includes>
+                </configuration>
+            </plugin>
+            <!-- Avoid build warning about generating empty jar -->
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-jar</id>
+                        <phase>none</phase>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>rpm</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <file>
+                    <exists>/usr/local/bin/rpm</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>rpm-maven-plugin</artifactId>
+                        <configuration>
+                            <group>Applications/Archiving</group>
+                            <mappings combine.self="override">
+                                <mapping>
+                                    <directory>/var/www/html/licenses</directory>
```

Comment:
I don't think we want to install or expose these licenses via this path.
In fact, I'm most certain this `pom.xml` file doesn't even need a `<profiles>`.